influxql/ast.go

```diff
@@ -839,6 +839,38 @@ func (s *SelectStatement) HasWildcard() bool {
 	return false
 }
 
+// hasCount returns whether or not the select statement has at least 1 count aggregate function
+func (s *SelectStatement) hasCount() bool {
+	for _, f := range s.Fields {
+		c, ok := f.Expr.(*Call)
+		if ok && strings.ToLower(c.Name) == "count" {
+			return true
+		}
+	}
+
+	return false
+}
+
+// hasWhereTime returns whether or not the select statement has at least 1
+// where condition with time as the condition
+func (s *SelectStatement) hasWhereTime(node Node) bool {
```

Comment:
Very similar to `OnlyTimeDimensions`, but not exactly the same. Can we consolidate the two functions? Perhaps not. If nothing else, can we standardize on the term "time dimension"?